### ISSUE TITLE
Status bar Hide when VideoPlaying

### ISSUE THREAD

**Issue by tonyawad88** [moust/cordova-plugin-videoplayer#48]
Hello,

Is there any chance to hide the status bar (clock, battery, notifications, etc) hile the video is playing? 

Thank you !

**Comment by tonyawad88** [comment]
Any chance of adding the following lines to your plugin please:
Method Name: **openVideoDialog**
```
//When you set up your dialog can you add the following line to make it go fullscreen?
dialog.getWindow().setFlags(LayoutParams.FLAG_FULLSCREEN, LayoutParams.FLAG_FULLSCREEN);
```

**Comment by moust** [comment]
Resolved with #49
